src/screens/ImportSeedPhraseSheet.js

```diff
@@ -225,7 +226,7 @@ const ImportSeedPhraseSheet = ({ isEmpty, setAppearListener }) => {
     if (!wasImporting && isImporting) {
       startAnalyticsTimeout(async () => {
         const input = resolvedAddress ? resolvedAddress : seedPhrase.trim();
-        initializeWallet(input, color, name || '')
+        initializeWallet(input, color, name ? name : '')
```

Comment:
And this one

Reply:
The same reason as above - the `||` syntax crash on android

Reply:
why it happens? `''` should be casted to `false`?

Reply:
Something seems to be working strangely. `name` can be sth (non-string) castable to `true`. let's debug it outside of the scope of this PR since it follows original logic.